string name
IntersectionReferenceID_t[] id
uint8 msgCnt
int8[16] statusSPAT
MovementList_t[] states
bool maneuverAssist
